DigitalLearningSolutions.Web/Views/Frameworks/Developer/AddCompetencyLearningResources.cshtml

```diff
@@ -37,7 +37,7 @@
       <div class="search-box-container" id="search">
         @Html.LabelFor(m => m.SearchText, "Search", new { @class = "nhsuk-u-visually-hidden", @for = "search-field" })
         @Html.TextBoxFor(m => m.SearchText, new { @class = "search-box", id = "search-field", name = "searchString", type = "search", placeholder = "Search", autocomplete = "off" })
-        <button class="nhsuk-search__submit sort-submit" type="submit">
+        <button class="nhsuk-search__submit sort-submit" type="submit" asp-route-page="1">
```

Comment:
asp-route-page="1" is working fine, here. The problem is the hidden input on line 46 below which needs removing.